### PR TITLE
CB-10440 Add CSS color names support for BackgroundColor on Windows

### PR DESCRIPTION
**Pull request by daserge**
[Jira issue](https://issues.apache.org/jira/browse/CB-10440)

### CODE REVIEW

**Comment by nikhilkh** [comment]
Could you add a unit test for this?

File: template/cordova/lib/AppxManifest.js [hunk 1]
```diff
@@ -374,6 +374,11 @@ AppxManifest.prototype.getCapabilities = function () {
 };
 
 function refineColor(color) {
+    if (color.indexOf('0x') === -1 && color.indexOf('#') === -1) {
```
Comment (dblotsky) [inline template/cordova/lib/AppxManifest.js]: Please put this condition into a function; that way it will be easier to debug and read in the future.

Comment (dblotsky) [inline template/cordova/lib/AppxManifest.js]: Is there a list of valid CSS colours? For example, `meow` and `12345` and are not colours, but they will get accepted here.

Comment (nikhilkh) [inline template/cordova/lib/AppxManifest.js]: WIndows SDK will do the validation and omit errors for invalid values and it's not for us to validate here.

**Comment by daserge** [comment]
@nikhilkh, @dblotsky, thanks for review, I've addressed the notes.

**Comment by dblotsky** [comment]
LGTM.

**Pull request merged**